config/dcp_test.go

```diff
@@ -323,3 +324,50 @@ func TestApplyDefaultMetadata(t *testing.T) {
 		t.Errorf("Metadata.Type is not set to expected value")
 	}
 }
+
+func TestDcpMode(t *testing.T) {
+	t.Run("it should return true when dcp mode finite", func(t *testing.T) {
+		// Arrange
+		dcp := &Dcp{
+			Dcp: ExternalDcp{
+				Mode: Finite,
+			},
+		}
+
+		// Act
+		isDcpModeFinite := dcp.IsDcpModeFinite()
+
+		// Assert
+		assert.True(t, isDcpModeFinite)
```

Comment:
Using `assert` causes getting `github.com/stretchr/testify` dependency to our go-dcp library. We don't want to get this dependecy so please refactor to use pure t.Error etc. (You can check other test too)